activeweb/src/main/java/org/javalite/activeweb/RequestDispatcher.java

```diff
@@ -205,18 +205,12 @@ public void doFilter(ServletRequest req, ServletResponse resp, FilterChain chain
                 return;
             }
 
-            if (route != null) {
```

Comment:
@faisal-hameed I would rather study this code, and also run a coverage report, to fully ensure this cannot happen before removing. It was written for some reason :)